### DISCUSSION TITLE
Macbook Pro M1 Pro Monterey  ABL / Logic Agridder - Not Compatible

### DISCUSSION THREAD

**Discussion by Heimke** [apohl79/audiogridder#945] [Q&A]
Hi,

Since it's my first discussion entry, I'd like to initially thank you so much for this amazing tool, I can't state how much it helped me.
I updated to the latest Beta 8 but also experienced the same with 7, and 6 lately where none of the versions, ARM, Binary, or 86x can start. the strange thing is that it only applies to the Agridder efx plugin, not the instrument one, which works perfectly.
I've tried using Rossetta and natively however without success

![image](https://user-images.githubusercontent.com/105432834/168073703-c797ec14-6487-480d-b8ef-7eb7e4191b41.png)

Wonder if anyone experienced the same? i also tried erasing everything related to the plugin and reinstalling, mixing and matching the binaries with no luck

I'll appreciate any advice 
I am on a Macbook Pro M1 Pro, 10 core, Monterey 12.3.1 running - Ableton live 11.1, Logic Pro X 10.7.4, Cubase Pro 12.0.2 (VST3 obviously)

Thank you!

**Comment by naokiapri** [comment]
i've had this error while using 41400 sample rate, at 48000 plugin is initialising normally.
